pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[tool.isort]
```

Comment:
Can you add a description to the PR on how to parse this file and what the different fields in this file indicate?

Reply:
Added some info. We should double check to see if we need `known_third_party`. Maintaining this list seems like it could become a lot of overhead.